modules/channel-web/src/views/lite/store/view.ts

```diff
@@ -152,6 +152,11 @@ class ViewStore {
     }
   }
 
+  @action.bound
+  postMessage(name: string) {
+    parent.postMessage({ name: name }, '*')
```

Comment:
Can you change it to window.parent ? It's more explicit, otherwise we may be looking for the parent variable